packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -10,21 +10,60 @@
 @import '~carbon-components/scss/globals/scss/layout';
 @import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
 @import '~carbon-components/scss/globals/scss/css--reset';
+@import '~carbon-components/scss/components/ui-shell/ui-shell';
+@import '~carbon-components/scss/components/ui-shell/side-nav';
 
-/// Masthead styles
 /// @access private
-/// @group masthead
+/// @group dotcom ui-shell
+
+$search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
+$search-transition-timing: 95ms;
+
 @mixin masthead {
   .#{$prefix}--header {
+    transition-timing-function: $search-transition;
     background: $ui-02;
```

Comment:
Any particular reason why we chose `$ui-02` instead of `$ui-background`?

Reply:
Ah, you are right.